rust/headless-client/src/known_dirs.rs

```diff
@@ -7,13 +7,19 @@
 //!
 //! I wanted the ProgramData folder on Windows, which `dirs` alone doesn't provide.
 
-pub use imp::{logs, runtime, session, settings};
+pub use platform::{ipc_service_logs, logs, runtime, session, settings};
 
 #[cfg(any(target_os = "linux", target_os = "macos"))]
-pub mod imp {
+pub mod platform {
     use connlib_shared::BUNDLE_ID;
     use std::path::PathBuf;
 
+    #[allow(clippy::unnecessary_wraps)]
+    pub fn ipc_service_logs() -> Option<PathBuf> {
+        // TODO: This is magic, it must match the systemd file
+        Some(PathBuf::from("/var/log").join(connlib_shared::BUNDLE_ID))
```

Comment:
I guess we don't run this one on MacOS so it is "fine" but technically, that is not valid, is it?

Reply:
Would it be better to just make a separate module for MacOS where all of these return `None` or `unimplemented!`?